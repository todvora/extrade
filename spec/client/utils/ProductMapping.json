[
  {
    "period": {
      "from": "1.1.2014",
      "till": "30.6.2014"
    },
    "direction": "import",
    "groupBy": "Q",
    "results": [
      {
        "period": "1/2014",
        "code": "87120030",
        "name": "Jízdní kola, bez motoru (kromě bez kuličkových ložisek)",
        "country": "AT",
        "countryName": "Rakousko",
        "weight": "19733",
        "price": "7332",
        "unit": "PCE",
        "count": "1427"
      },
      {
        "period": "1/2014",
        "code": "87120030",
        "name": "Jízdní kola, bez motoru (kromě bez kuličkových ložisek)",
        "country": "DE",
        "countryName": "Německo",
        "weight": "51719",
        "price": "65311",
        "unit": "PCE",
        "count": "4186"
      },
      {
        "period": "1/2014",
        "code": "87149110",
        "name": "Rámy pro vozidla čísel 8711 až 8713 (kromě pro motocykly,mopedy a vozí",
        "country": "DE",
        "countryName": "Německo",
        "weight": "26",
        "price": "189",
        "unit": "PCE",
        "count": "64"
      },
      {
        "period": "2/2014",
        "code": "87120030",
        "name": "Jízdní kola, bez motoru (kromě bez kuličkových ložisek)",
        "country": "AT",
        "countryName": "Rakousko",
        "weight": "5628",
        "price": "3544",
        "unit": "PCE",
        "count": "380"
      },
      {
        "period": "2/2014",
        "code": "87120030",
        "name": "Jízdní kola, bez motoru (kromě bez kuličkových ložisek)",
        "country": "DE",
        "countryName": "Německo",
        "weight": "59664",
        "price": "49284",
        "unit": "PCE",
        "count": "4716"
      },
      {
        "period": "2/2014",
        "code": "87149110",
        "name": "Rámy pro vozidla čísel 8711 až 8713 (kromě pro motocykly,mopedy a vozí",
        "country": "DE",
        "countryName": "Německo",
        "weight": "37",
        "price": "294",
        "unit": "PCE",
        "count": "48"
      }
    ]
  },
  {
    "period": {
      "from": "1.1.2014",
      "till": "30.6.2014"
    },
    "direction": "export",
    "groupBy": "Q",
    "results": [
      {
        "period": "1/2014",
        "code": "87120030",
        "name": "Jízdní kola, bez motoru (kromě bez kuličkových ložisek)",
        "country": "AT",
        "countryName": "Rakousko",
        "weight": "460302",
        "price": "125922",
        "unit": "PCE",
        "count": "24207"
      },
      {
        "period": "1/2014",
        "code": "87120030",
        "name": "Jízdní kola, bez motoru (kromě bez kuličkových ložisek)",
        "country": "DE",
        "countryName": "Německo",
        "weight": "550446",
        "price": "195410",
        "unit": "PCE",
        "count": "37097"
      },
      {
        "period": "1/2014",
        "code": "87149110",
        "name": "Rámy pro vozidla čísel 8711 až 8713 (kromě pro motocykly,mopedy a vozí",
        "country": "AT",
        "countryName": "Rakousko",
        "weight": "568",
        "price": "857",
        "unit": "PCE",
        "count": "377"
      },
      {
        "period": "1/2014",
        "code": "87149110",
        "name": "Rámy pro vozidla čísel 8711 až 8713 (kromě pro motocykly,mopedy a vozí",
        "country": "DE",
        "countryName": "Německo",
        "weight": "5937",
        "price": "4394",
        "unit": "PCE",
        "count": "1180"
      },
      {
        "period": "2/2014",
        "code": "87120030",
        "name": "Jízdní kola, bez motoru (kromě bez kuličkových ložisek)",
        "country": "AT",
        "countryName": "Rakousko",
        "weight": "304159",
        "price": "85356",
        "unit": "PCE",
        "count": "15368"
      },
      {
        "period": "2/2014",
        "code": "87120030",
        "name": "Jízdní kola, bez motoru (kromě bez kuličkových ložisek)",
        "country": "DE",
        "countryName": "Německo",
        "weight": "614751",
        "price": "198422",
        "unit": "PCE",
        "count": "41406"
      },
      {
        "period": "2/2014",
        "code": "87149110",
        "name": "Rámy pro vozidla čísel 8711 až 8713 (kromě pro motocykly,mopedy a vozí",
        "country": "AT",
        "countryName": "Rakousko",
        "weight": "290",
        "price": "454",
        "unit": "PCE",
        "count": "120"
      },
      {
        "period": "2/2014",
        "code": "87149110",
        "name": "Rámy pro vozidla čísel 8711 až 8713 (kromě pro motocykly,mopedy a vozí",
        "country": "DE",
        "countryName": "Německo",
        "weight": "22164",
        "price": "9315",
        "unit": "PCE",
        "count": "9118"
      }
    ]
  }
]