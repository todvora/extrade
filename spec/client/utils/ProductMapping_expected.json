[
  {
    "code": "87120030",
    "name": "Jízdní kola, bez motoru (kromě bez kuličkových ložisek)",
    "unit": "PCE",
    "intervals": [
      {
        "period": "1/2014",
        "import": [
          {
            "country": "AT",
            "countryName": "Rakousko",
            "weight": "19733",
            "price": "7332",
            "count": "1427"
          },
          {
            "country": "DE",
            "countryName": "Německo",
            "weight": "51719",
            "price": "65311",
            "count": "4186"
          }
        ],
        "export": [
          {
            "country": "AT",
            "countryName": "Rakousko",
            "weight": "460302",
            "price": "125922",
            "count": "24207"
          },
          {
            "country": "DE",
            "countryName": "Německo",
            "weight": "550446",
            "price": "195410",
            "count": "37097"
          }
        ]
      },
      {
        "period": "2/2014",
        "import": [
          {
            "country": "AT",
            "countryName": "Rakousko",
            "weight": "5628",
            "price": "3544",
            "count": "380"
          },
          {
            "country": "DE",
            "countryName": "Německo",
            "weight": "59664",
            "price": "49284",
            "count": "4716"
          }
        ],
        "export": [
          {
            "country": "AT",
            "countryName": "Rakousko",
            "weight": "304159",
            "price": "85356",
            "count": "15368"
          },
          {
            "country": "DE",
            "countryName": "Německo",
            "weight": "614751",
            "price": "198422",
            "count": "41406"
          }
        ]
      }
    ]
  },
  {
    "code": "87149110",
    "name": "Rámy pro vozidla čísel 8711 až 8713 (kromě pro motocykly,mopedy a vozí",
    "unit": "PCE",
    "intervals": [
      {
        "period": "1/2014",
        "import": [
          {
            "country": "DE",
            "countryName": "Německo",
            "weight": "26",
            "price": "189",
            "count": "64"
          }
        ],
        "export": [
          {
            "country": "AT",
            "countryName": "Rakousko",
            "weight": "568",
            "price": "857",
            "count": "377"
          },
          {
            "country": "DE",
            "countryName": "Německo",
            "weight": "5937",
            "price": "4394",
            "count": "1180"
          }
        ]
      },
      {
        "period": "2/2014",
        "import": [
          {
            "country": "DE",
            "countryName": "Německo",
            "weight": "37",
            "price": "294",
            "count": "48"
          }
        ],
        "export": [
          {
            "country": "AT",
            "countryName": "Rakousko",
            "weight": "290",
            "price": "454",
            "count": "120"
          },
          {
            "country": "DE",
            "countryName": "Německo",
            "weight": "22164",
            "price": "9315",
            "count": "9118"
          }
        ]
      }
    ]
  }
]
